_summaries/liu2019large-scale_2.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: liu2019large-scale
+author: Shivi Agarwal
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of this paper?
+Real world data is long tailed and open-ended distributed. A practical recognition system must classify among majority and minority classes. 
+This paper discusses about the Open Long-Tailed Recognition (OLTR) as learning from naturally distributed data and optimizing the classification 
+accuracy over a balanced test set including head, tail, and open classes. OLTR focuses on imbalanced classification, few-shot learning, and open-set 
+recognition in single integrated algorithm, however the existing classification approaches focus only on one aspect and deliver poorly. The key 
+challenges for OLTR are tail recognition robustness and open-set sensitivity. The core idea of this paper is to develop an integrated OLTR algorithm 
```

Comment:
Not something you necessarily have to add to this summary but might be good to think about for the discussion: This paper trys to tackle a large problem in 1 shot. What is the motivation for this as opposed to using multiple models that are optimized for their own tasks (head, tail, out).

Reply:
Thankyou for the review. I believe adding these points can give clearer picture of the core idea of this paper. I will definitely agree with you at the time of discussion.